add step using "shadow databases" to prisma guide (#11169)
supabase/supabase: apps/docs/pages/guides/integrations/prisma.mdx

### Configuring the project to use PostgreSQL

> Prisma init migrations will try to drop `postgres` database by default, and this may lead to conflicts with Supabase initial migrations that are critically needed for the correct functioning of our managed databases. For such cases Prisma has a feature called [Shadow Databases](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).

Reviewer: The first sentence is a bit confusing so my suggestion here might not have the same meaning you were trying to get across.
```suggestion
By default, Prisma migrations will try to drop the `postgres` database, which can lead to conflicts with Supabase databases. For this scenario, use [Prisma Shadow Databases](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).
```

> Create a shadow database with the local database (or create another database in your PostgreSQL server with the same Supabase project) using the `psql` CLI and the `DATABASE_URL` from the previous steps.

Author: ```suggestion
Create a shadow database in your PostgreSQL server within the same Supabase project using the `psql` CLI and the `DATABASE_URL` from the previous steps (or use the local database).
```

Author: Thanks for the fixes!
I think the first sentence is quite good now.
Wdyt about the second one? I have a feeling that it would be more correct, but not sure about my wording

Reviewer: LGTM!
